vtr_flow/scripts/python_libs/vtr/inspect.py

```diff
@@ -0,0 +1,198 @@
+import re
+from collections import OrderedDict
+
+try:
+    #Try for the fast c-based version first
+    import xml.etree.cElementTree as ET
+except ImportError:
+    #Fall back on python implementation
+    import xml.etree.ElementTree as ET
+
+from vtr import load_config_lines
+from vtr.error import InspectError
+
+class ParsePattern:
+    def __init__(self, name, filename, regex_str, default_value=None):
+        self._name = name
+        self._filename = filename
+        self._regex = re.compile(regex_str)
+        self._default_value = default_value
+
+    def name(self):
+        return self._name
+
+    def filename(self):
+        return self._filename
+
+    def regex(self):
+        return self._regex
+
+    def default_value(self):
+        return self._default_value
+
+class PassRequirement(object):
+    def __init__(self, metric):
+        self._metric = metric
+        self._type = type
+
+    def metric(self):
+        return self._metric
+
+    def type(self):
+        raise NotImplementedError()
+
+    def check_passed(golden_value, check_value):
+        raise NotImplementedError()
+
+class EqualPassRequirement(PassRequirement):
+    def __init__(self, metric):
+        super(EqualPassRequirement, self).__init__(metric)
```

Comment:
Should use python3 super format, ie: super().__init__(args)

This occurs in many places.

Reply:
This has been fixed